A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_Elec.js

```diff
@@ -0,0 +1,1081 @@
+            this.hide(this.e_EMERGEN_VOLTS_VALUE)
+            this.hide(this.e_EMERGEN_VOLTS_UNIT)
+            this.hide(this.e_EMERGEN_FREQ_VALUE)
+            this.hide(this.e_EMERGEN_FREQ_UNIT)
+        }
+
+        setACESSBUS_SHED_ON() {
+            this.show(this.e_ACESSBUS_SHED)
+        }
+        setACESSBUS_SHED_OFF() {
+            this.hide(this.e_ACESSBUS_SHED)
+        }
+
+        setIDG1_LOPR_ON() {
+            this.show(this.e_IDG1_LOPR)
+        }
+        setIDG1_LOPR_OFF() {
+            this.hide(this.e_IDG1_LOPR)
+        }
+
+        setIDG2_LOPR_ON() {
+            this.show(this.e_IDG2_LOPR)
+        }
+        setIDG2_LOPR_OFF() {
+            this.hide(this.e_IDG2_LOPR)
+        }
+
+        setIDG1_DISC_ON() {
+            this.show(this.e_IDG1_DISC)
+        }
+        setIDG1_DISC_OFF() {
+            this.hide(this.e_IDG1_DISC)
+        }
+
+        setIDG2_DISC_ON() {
+            this.show(this.e_IDG2_DISC)
+        }
+        setIDG2_DISC_OFF() {
+            this.hide(this.e_IDG2_DISC)
+        }
+
+        setDCESSBUS_SHED_ON() {
+            this.show(this.e_DCESSBUS_SHED)
+        }
+        setDCESSBUS_SHED_OFF() {
+            this.hide(this.e_DCESSBUS_SHED)
+        }
+
+        setGALLEY_SHED_ON() {
+            this.show(this.e_GALLEY_SHED_TOP)
+            this.show(this.e_GALLEY_SHED_BOTTOM)
+        }
+        setGALLEY_SHED_OFF() {
+            this.hide(this.e_GALLEY_SHED_TOP)
+            this.hide(this.e_GALLEY_SHED_BOTTOM)
+        }
+
+        setSTATINV_ON() {
+            this.show(this.e_STATINV_TITLE)
+            this.show(this.e_ARROW_BAT1_STATINV)
+        }
+        setSTATINV_OFF() {
+            this.hide(this.e_STATINV_TITLE)
+            this.hide(this.e_ARROW_BAT1_STATINV)
+        }
+
+        setCONNECTION_DCBAT_BAT1_ON() {
+            this.show(this.e_WIRE_DCBAT_BAT1)
+            this.show(this.e_WIRE_BAT1_DCBAT)
+        }
+        setCONNECTION_DCBAT_BAT1_OFF() {
+            this.hide(this.e_WIRE_DCBAT_BAT1)
+            this.hide(this.e_WIRE_BAT1_DCBAT)
+        }
+
+        setCONNECTION_DCBAT_BAT2_ON() {
+            this.show(this.e_WIRE_DCBAT_BAT2)
+            this.show(this.e_WIRE_BAT2_DCBAT)
+        }
+        setCONNECTION_DCBAT_BAT2_OFF() {
+            this.hide(this.e_WIRE_DCBAT_BAT2)
+            this.hide(this.e_WIRE_BAT2_DCBAT)
+        }
+
+        setCONNECTION_DCBAT_BAT1_CHARGE_ON() {
+            this.show(this.e_WIRE_DCBAT_BAT1)
+            this.show(this.e_ARROW_DCBAT_BAT1)
+        }
+        setCONNECTION_DCBAT_BAT1_CHARGE_OFF() {
+            this.hide(this.e_WIRE_DCBAT_BAT1)
+            this.hide(this.e_ARROW_DCBAT_BAT1)
+        }
+
+        setCONNECTION_DCBAT_BAT2_CHARGE_ON() {
+            this.show(this.e_WIRE_DCBAT_BAT2)
+            this.show(this.e_ARROW_DCBAT_BAT2)
+        }
+        setCONNECTION_DCBAT_BAT2_CHARGE_OFF() {
+            this.hide(this.e_WIRE_DCBAT_BAT2)
+            this.hide(this.e_ARROW_DCBAT_BAT2)
+        }
```

Comment:
You should rather have a function like `setCONNECTION_DCBAT_BAT2_CHARGE` with a boolean parameter that represents the on/off state. This is quite boilerplate-y right now.